gatsby-config.js

```diff
@@ -80,7 +82,7 @@ module.exports = {
         ],
       },
     },
-
+    "@fec/gatsby-plugin-advanced-feed",
```

Comment:
Why this plugin versus the [officially supported Gatsby version](https://github.com/gatsbyjs/gatsby/tree/master/packages/gatsby-plugin-feed)?

Reply:
> Why this plugin versus the [officially supported Gatsby version](https://github.com/gatsbyjs/gatsby/tree/master/packages/gatsby-plugin-feed)?

Because this one supports the Atom feed and RSS feed. The official one does not support the Atom. Do you think I should use the official one, and I transfer the format from RSS to Atom? I google them, it seems RSS and Atom are very similar.